### ISSUE TITLE
Issue with latex equations start and end symbols registering in OCR

### ISSUE THREAD

**Issue by nikedinikedi** [supermemo/SuperMemoAssistant.Plugins.LateX#12]
### New description

When OCRing both Text and Formulas (math, chemistry, ...), add a feature to automatically convert LaTeX Math embeds `\[` and `\]` as SMA tags `[$]` and `[/$]`.

See below for illustration.

### Old description

When OCR'ing chunk of certain layout with several latex symbols or equations, the equation isolated/seperated/in between two paragraphs will miss [$] and [/$] 

![image](https://user-images.githubusercontent.com/51511831/77255943-b1e78380-6c73-11ea-8d0a-06ef1ae3a7fc.png)

---------

This is what i OCR'd (the whole thing)

![image](https://user-images.githubusercontent.com/51511831/77256032-4fdb4e00-6c74-11ea-92e1-2ab5341f0441.png)

-------------

This has happened with multiple similiar layouts when i OCR such chunk.
Note: if i OCR this equation individually, it works fine.

Here's the page of this PDF (the bug occurs with the equation in bottom)

[apstats2.pdf](https://github.com/supermemo/SuperMemoAssistant.Plugins.LateX/files/4365689/apstats2.pdf)

**Comment by nikedinikedi** [comment]
Similiar case with little different settings, but the main idea is same, when there's line break this bug happens:

Image 1. What i OCR'd
![image.png](https://images.zenhubusercontent.com/5e489b218f780f395af8e838/a27ba3f9-1957-4cf2-8804-84cf243ef3c2)

Image 2. Tex Editor

![image.png](https://images.zenhubusercontent.com/5e489b218f780f395af8e838/df6fe9a4-61cb-409e-b2e2-db3d9a4bf44e)

**Comment by alexis-** [comment]
Doesn't seem to happen anymore 

![image.png](https://images.zenhubusercontent.com/57dfca7be40e5714b16f1384/023fb8be-aac4-46f4-9608-56de27595a51)

**Comment by nikedinikedi** [comment]
Try on longer, similar pieces as in the image

**Comment by nikedinikedi** [comment]
This is the main reason i don't use latex that often
-Naess

**Comment by alexis-** [comment]
I'll try the second PDF, but the test I shared was executed on the formula of the first PDF that you shared.

**Comment by alexis-** [comment]
Also we're running different versions of the PDF. I don't recall exactly all the changes, but I think I changed something in the OCR a while back.

**Comment by alexis-** [comment]
Ah wait, I see what you're doing here. You're OCRing everything, not just the formula. It's not meant to work like that. MathPix returns the LaTeX code for the whole image, meaning it thinks you also want the text to be part of the LaTeX document.

**Comment by nikedinikedi** [comment]
You need to include text not only the equation

**Comment by nikedinikedi** [comment]
Yes

**Comment by nikedinikedi** [comment]
It's inefficient to go one equation at once esp in math heavy papers jumping between words and equations. It works fine usually but not when there is a break like in the images

**Comment by alexis-** [comment]
> It's inefficient to go one equation at once esp in math heavy papers jumping between words and equations

I agree. It's not a bug though, it's a feature request as current functionality is working as expected.

**Comment by alexis-** [comment]
Done, will be in next version. Look out for any bugs.

**Comment by nikedinikedi** [comment]
BEAUTIFUL
